packages/idos-sdk-js/src/lib/enclave.ts

```diff
@@ -43,7 +43,10 @@ export class Enclave {
     const { encryptionPublicKey, humanId, signerAddress, signerPublicKey } =
       await this.provider.load();
 
-    this.idOS.store.set("encryption-public-key", encryptionPublicKey);
+    this.idOS.store.set(
+      "encryption-public-key",
+      encryptionPublicKey && Base64Codec.encode(encryptionPublicKey)
+    );
```

Comment:
I don't see us reading this, but I saw it being wrongly serialized without being in base64. So, I fixed it.

Unsure if perhaps a better idea would be to completely yank this (and other values we write but don't read) out.

Reply:
Saw this working out this change. But I think we should tidy this up. Resolving.